eng/Version.Details.xml

```diff
@@ -18,5 +18,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>7d0933a732a7156decd1f03cefe0d875a5a9954d</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.XUnitExtensions" Version="2.4.0-beta.18506.1">
```

Comment:
We should have considered resetting the version with the move to arcade and its name change but we are now over that point.

Reply:
I guess this will be updated automatically with DARC, I don't know how the arcade dependency flow works yet.

Reply:
correct.